.env.development

```diff
@@ -59,8 +59,10 @@ VITE_GNOSIS_NODE_URL=https://dev-daemon.gnosis.shapeshift.com
 VITE_ARBITRUM_NODE_URL=https://dev-api.arbitrum.shapeshift.com/api/v1/jsonrpc
 VITE_ARBITRUM_NOVA_NODE_URL=https://dev-api.arbitrum-nova.shapeshift.com/api/v1/jsonrpc
 VITE_BASE_NODE_URL=https://dev-api.base.shapeshift.com/api/v1/jsonrpc
-VITE_THORCHAIN_NODE_URL=https://dev-daemon.thorchain.shapeshift.com
-VITE_MAYACHAIN_NODE_URL=https://dev-daemon.mayachain.shapeshift.com
+# Swap me back to 9R as-needed
+# VITE_THORCHAIN_NODE_URL=https://thornode.ninerealms.com
+VITE_THORCHAIN_NODE_URL=https://dev-daemon.thorchain.shapeshift.com/lcd
```

Comment:
Cherry-picked https://github.com/shapeshift/web/pull/9450 - by the time you review this, it should either be merged, or it may not in which case we can interactive drop the cherry-pick here, not required for this PR to work, especially since it's only really using mock data for now